src/main/kotlin/sg/com/quantai/etl/ETLApplication.kt

```diff
@@ -1,13 +1,33 @@
 package sg.com.quantai.etl
 
+import org.springframework.context.annotation.Bean
+import org.springframework.web.client.RestTemplate
+import io.github.cdimascio.dotenv.dotenv
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
-import org.springframework.scheduling.annotation.EnableScheduling
 
 @SpringBootApplication
-@EnableScheduling
-class ETLApplication
+class ETLApplication {
+
+	@Bean
+	fun restTemplate(): RestTemplate {
+		return RestTemplate()
+	}
+}
 
 fun main(args: Array<String>) {
+
+	// Load environment variables from .env
+	val dotenv = dotenv {
+		ignoreIfMalformed = true // Ignore errors if the .env file is malformed
+		ignoreIfMissing = true    // Ignore errors if the .env file is missing
+	}
+
+	// Access the API key from .env
+	val cryptoCompareApiKey = dotenv["CRYPTOCOMPARE_API_KEY"]
```

Comment:
A Springboot application already has an environment file in terms of the properties file (e.g., src/main/resources/application.properties).

Using multiple environment files is, then, redundant and confusing.

Reply:
Removed
Added API key onto application.properties instead